charts/guard/values.yaml

```
@@ -0,0 +1,118 @@
+fullnameOverride: guard
+global:
+  # domain is the 
+  domain: path.grove.city  
+  # namespace is the namespace that both Envoy Gateway and the PATH service are deployed to.
+  namespace: path-local 
+  # serviceName is the name of the service that the PATH service is deployed to.
+  serviceName: path-http 
+
+  # Port is the port that the PATH service runs on in the cluster.
+  # This is the port that Envoy Gateway will forward requests to.
+  port: 3069
+
+# ---- GUARD configuration ----
+guard:
+  # Deploy K8s Gateway resource. Enabled by default
+  gateway:
+    enabled: true
+    port: 3070
+
+  # ---- Service routing settings ----
+
+  # List of services that will be routed by Envoy Gateway
+  # to the PATH backend. These services will be used to
+  # construct HTTPRoutes for each service. These HTTPRoutes
+  # are used to assign the "target-service-id" header to the
+  # request based on the subdomain service alias.
+  # - serviceId [REQUIRED]: The authorative service ID of the service.
+  # - aliases [OPTIONAL]: A list of aliases for the service.
+  #
+  # For example:
+  # - F00C.path.grove.city -> "target-service-id: F00C"
+  # - eth.path.grove.city -> "target-service-id: F00C"
+  # - polygon.path.grove.city -> "target-service-id: F021"
+  #
+  # IMPORTANT: For production deployments, the `services` field
+  # should be overridden with the actual services that will be
+  # routed to the PATH backend using a `guard-values.yaml` file.
+  services:
+    - serviceId: F00C
+      aliases:
+        - eth
+        - eth-mainnet
+    - serviceId: F021
+      aliases:
+        - polygon
+        - polygon-mainnet
+
+# ---- External Auth Server configuration ----
+authServer:
+  enabled: true
+  replicas: 1
+  port: 10001
+  env:
+    - name: PORT
+      value: "10001"
+    - name: GRPC_HOST_PORT
+      value: guard-pads:10002
+    - name: GRPC_USE_INSECURE_CREDENTIALS
+      value: "true"
+
+# ---- PADS configuration ----
+pads:
+  enabled: true
+  replicas: 1
+  port: 10002
+  # IMPORTANT: Exactly one of the following sections below must be set:
+  # - 1. YAML_FILEPATH
+  # - 2. POSTGRES_CONNECTION_STRING
+  # An error will be thrown if neither or both are set.
+
+  # -- 1. YAML_FILEPATH --
+  # IMPORTANT: To use a Postgres database as the PADS data source, comment out
+  # the below section and use the POSTGRES_CONNECTION_STRING section instead.
+  #
+  # If the YAML_FILEPATH environment variable is set, 
+  # PADS will use the file at the specified path as the data source.
+
+  env:
+  - name: PORT
+    value: "10002"
+  - name: YAML_FILEPATH
+    value: "/app/data/.gateway-endpoints.yaml"
+  # Gateway endpoints define the allowed endpoints for a PATH instance.
+  # For a detailed breakdown of the possible format of this file, see:
+  # - https://path.grove.city/develop/envoy/envoy_config#4-gateway-endpoints-data---gateway-endpointsyaml
+  # - https://path.grove.city/develop/envoy/walkthrough#gateway-endpoint-authorization
```

This is optional. I've found that section headers are a lot more dynamic and likely to change

```suggestion
  # - See gateway endpoints yamla at https://path.grove.city/develop/envoy/envoy_config 
  # - See endpoint authorization at https://path.grove.city/develop/envoy/walkthrough
```